src/core.c/CompUnit/RepositoryRegistry.pm6

```diff
@@ -80,6 +80,10 @@ class CompUnit::RepositoryRegistry {
                 nqp::push($raw-specs,$_)
                   for parse-include-specS(nqp::atkey($ENV,'RAKUDOLIB'));
             }
+            if nqp::existskey($ENV,'RAKULIB') {
+                nqp::push($raw-specs,$_)
+                  for parse-include-specS(nqp::atkey($ENV,'RAKULIB'));
+            }
```

Comment:
It should be 'either-either', not both. I.e. if `RAKULIB` is defined it takes over `PERL6LIB`. This would allow older and newer versions of rakudo to co-exists peacefully whenever such functionality is needed.

Reply:
I'll put an 'elsif' here testing for RAKULIB first.

Reply:
I think RAKUDOLIB and RAKULIB are supposed to work cumulatively.

Reply:
If I know right, both PERL6LIB and RAKUDOLIB are to be deprecated. This logic is there for backward compatibility only. I fear that cumulating might encourage ways of use that we don't want to support in the long run.